desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/steps/seller/SellerStep3View.java

```diff
@@ -475,14 +487,60 @@ else if (paymentAccountPayload instanceof AmazonGiftCardAccountPayload)
 
     private void confirmPaymentReceived() {
         log.info("User pressed the [Confirm payment receipt] button for Trade {}", trade.getShortId());
-        busyAnimation.play();
-        statusLabel.setText(Res.get("shared.sendingConfirmation"));
 
-        model.dataModel.onFiatPaymentReceived(() -> {
-        }, errorMessage -> {
-            busyAnimation.stop();
-            new Popup().warning(Res.get("popup.warning.sendMsgFailed")).show();
-        });
+        model.dataModel.setSellerConfirmedPaymentReceiptDate();
+        updateConfirmButton();
+        if (!model.dataModel.requiresPayoutDelay() || model.dataModel.requiredPayoutDelayHasPassed()) {
+            onReleaseBitcoin();
+        }
+    }
+
+    private void updateConfirmButton() {
+        if (model.dataModel.requiresPayoutDelay() &&
+                model.dataModel.getSellerConfirmedPaymentReceiptDate() > 0) {
+            if (payoutDelayTimer == null) {
+                confirmButton.setText(Res.get("portfolio.pending.step3_seller.releaseBitcoin"));
+                confirmButton.setOnAction(e -> onReleaseBitcoin());
+                payoutDelayTimer = UserThread.runPeriodically(this::updateConfirmButtonDisableState, 1, TimeUnit.SECONDS);
+            }
+            if (!model.dataModel.requiredPayoutDelayHasPassed()) {
+                new Popup().warning(Res.get("portfolio.pending.step3_seller.delayedPayout",
```

Comment:
This pops up every time the view is activated. I think it should only pop up once after confirming payment receipt.

Reply:
I preferred to do it that way as we do not show additional info in the UI. Users who quickly close without reading (they exist, I am such one ;-)) might get lost otherwise. Alternatively we could add a "dont show again" checkbox per trade.

Reply:
Fair point on clicking away popups, don't show again, per trade, sounds like a reasonable compromise.

Reply:
I added the dont show again checkbox